src/sections/subSahara.html

```diff
@@ -8,52 +8,54 @@
+                <p class="font-light text-xs text-white">kurze Einführung über 4-5 Zeilen At vero eos et accusam et justo duo
+                    dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L vero eos et accusam et justo
+                    duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L</p>
             </div>
-            <div class="pr-36 lg:absolute lg:right-0  lg:bottom-0">
-                <img class="" src="../../images/Africa.webp" alt="" width="450">
+            <div class="my-20 flex flex-col lg:flex-row gap-8 sm:relative lg:relative">
+                <div class=" ojemba-gradient2 lg:w-4/5  p-6 lg:p-24 flex flex-col gap-3  ">
+                    <p class="font-railwayBold text-white text-xs">Ein <span class="border-b-2 border-green">stetig wachsender
+                        </span> Markt und Talentpool</p>
+                    <ul class="text-white text-xs flex flex-col gap-3">
+                        <li class="before:content-['•']"> durchschnittliches jährliches BIP-Wachstum von 7,2 <br> Prozent (2021) (The World Bank, 2022)</li>
+        
+                        <li class="before:content-['•']"> Einwohnerzahl: 13,46 Millionen (2021) prognostizierte <br> Wachstumrate von 1,4 % pro Jahr bis 2050</li>
+        
+                        <li class="before:content-['•']"> ~70% der Einwohner*innen sind unter 30 Jahre</li>
+        
+                        <li class="before:content-['•']"> wachsender Markt an bilingual ausgebildeten <br> Arbeitskräften </li>
+        
+                        <li class="before:content-['•']"> Größer Frauenanteil (61 %) im Parlament weltweit</li>
+                    </ul>
+                </div>
+                <div class="pr-36 lg:absolute lg:right-0 lg:bottom-0">
+                    <img class="" src="../../images/Africa.webp" alt="" width="450">
+                </div>
             </div>
         </div>
-    </div>
-    
-<footer class="flex lg:flex-row flex-col  justify-between items-center lg:pr-48 h-80 p-2  lg:pl-36 " style="background-image: url(../../images/bg_hero-1.webp)">
-    <div class="flex gap-10">
-<img src="../../images/people.svg" width="90" alt="people" >
-<p class="font-railwayBold text-xs text-white">stetig wachsender Markt und <br> Talentpool</p>
-    </div>
-    <div class="flex gap-10">
-        <img src="../../images/watch.svg" width="90"  alt="people" >
-<p class="font-railwayBold text-xs text-white">sehr gutes <br> Investitionsklima</p>
-    </div>
-    <div class="flex gap-10">
-        <img src="../../images/tick.svg" width="90"  alt="people" >
-<p class="font-railwayBold text-xs text-white">ausgezeichnete Infrastruktur <br> und hoher Digitalisierungsgrad</p>
-    </div>
-</footer>
-</section>
+        
+    <footer class="flex lg:flex-row flex-col  justify-between items-center lg:pr-48 h-80 p-2 xl:bg-no-repeat lg:pl-36 " style="background-image: url(../../images/bg_hero-1.webp)">
```

Comment:
below the footer there is no background color of blue, it's just oblique line below the linear gradient.


![image](https://github.com/divinecharlotte/north-star-team7/assets/81089765/92c86b1b-f873-4f10-bc32-c0d63aba1260)